index.html

```diff
@@ -90,6 +87,42 @@ <h3 class="projectname">Project name goes here</h3>
         </div>
     </section>
     <section class="aboutme">
+        <section class="headline">
+            <h1>About</h1>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+            <button type="button" class="about-button">Get my CV<img src="img/download.png" alt="Download"></button>  
+        </section>
+        <section class="tech">
+            <div class="languages1">
+                <h4>Languages</h4>
+                <ul class="position">
+                    <li>JavaScript (ES5/ES6)</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+            <div class="frameworks">
+                <h4>Frameworks</h4>
+                <ul class="position">
+                    <li>Bootstrap</li>
+                    <li>Ruby on rails</li>
+                    <li>RSpec</li>
+                    <li>Capibara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+            <div class="skills">
```

Comment:
- [ ] Great job so far 👍 The technology categories ("Languages", "Frameworks" and "Skills") should also be in a `ul` tag, each of them in a `li`. Kindly implement this functionality and you'll be good to go 👍

Reply:
Hi @Kingstalux 
The thing is that on the template there are different blocks. For this, do I need to make a ul tag inside another ul tag? 
I didn't understand this required change, can you please clarify for me?
Regards.

Reply:
Your going to have a list that contains the categories, then in the li for languages, you'll have another lists for everything under languages, then under the li for Frameworks you'll have another list again. So basically you'll have 2 sub ul lists inside the main ul list